src/combineTests.ts

```diff
@@ -79,16 +81,17 @@ export function shortenClassNames(options: {imports: string[]; body: string}) {
 export function combineTests(options:
     {
       existingImports?: string[];
-      tests: Array<{imports: string[]; staticImports: string[]; test?: string; testBody?: string; body?: string; }>;
+      tests: ILooseTestData[];
     },
 ) {
   const tests = options.tests;
+  const standardisedTests: ITestData[] = standardiseTests(tests);
```

Comment:
It should infer the return type automatically:

```suggestion
  const standardisedTests = standardiseTests(tests);
```